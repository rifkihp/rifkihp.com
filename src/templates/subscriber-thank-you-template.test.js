import React from 'react';
import renderer from 'react-test-renderer';

import SubscriberThankYouTemplate from './subscriber-thank-you-template';

describe('SubscriberThankYouTemplate', () => {
  const props = {
    data: {
      site: {
        siteMetadata: {
          title: 'test',
          url: 'test',
        },
      },
    },
  };

  it('renders correctly', () => {
    const tree = renderer.create(<SubscriberThankYouTemplate {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
