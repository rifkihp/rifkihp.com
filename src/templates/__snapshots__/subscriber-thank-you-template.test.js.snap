// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscriberThankYouTemplate renders correctly 1`] = `
<StaticQuery
  render={[Function]}
/>
`;
